migrations/900-websites-status-and-disabled.sql

```diff
@@ -0,0 +1,27 @@
+ALTER TABLE `websites_website` ADD COLUMN `status` integer UNSIGNED NOT NULL, ADD COLUMN `is_disabled` bool NOT NULL;
+
+CREATE TABLE `websites_websitepopularity` (
```

Comment:
These table names don't seem ideal? Can we do `websites_popularity` and `websites_trending`?